apps/docs/app/features/code-block/CodeBlock.tsx

```diff
@@ -59,32 +72,51 @@ export const CodeBlockContainer = ({
       zIndex="0"
       maxWidth="calc(100vw - var(--spor-space-4))"
       __css={{ "pre > div": { whiteSpace: "initial", maxWidth: "100%" } }}
+      onKeyUp={handleKeyUp}
+      ref={containerRef}
       {...props}
     >
       <Box width="100%" overflowX="hidden">
         {children}
       </Box>
-      <CopyCodeButton code={code} />
+      <CopyCodeButton ref={copyButtonRef} code={code} />
     </Box>
   );
 };
 
-type CopyCodeButtonProps = { code: string };
-export const CopyCodeButton = ({ code }: CopyCodeButtonProps) => {
-  const { onCopy, hasCopied } = useClipboard(code);
-  return (
-    <DarkMode>
-      <Button
-        variant="additional"
-        size="xs"
-        onClick={onCopy}
-        position="absolute"
-        top={2}
-        right={2}
-        fontFamily="body"
-      >
-        {hasCopied ? "Kopiert" : "Kopiér"}
-      </Button>
-    </DarkMode>
+function getPreviousFocusableElement() {
+  const allTabbableElements = Array.from(
+    document.querySelectorAll(
+      'a, button, input, select, textarea, [contenteditable], [tabindex]:not([tabindex^="-"])'
```

Comment:
Note - dette er ikke en komplett liste over alle fokuserbare items. Det tar heller ikke høyde for tabindex-rekkefølgen. Men det er en best-yet-low effort implementasjon. 

Kanskje en fremtidig Trello-lapp? 😁